src/components/Layout/Layout.module.css

```diff
@@ -0,0 +1,87 @@
+.newLayout {
+  --black: lab(0.07 0.35 0.12);
+  --dark-grey: lab(43.19 0 0);
+  --grey: lab(66.99 0 0);
+  --light-grey: lab(94.10 0 0);
+  --off-white: lab(100 0 0);
+
+  --blue: lab(49.827 31.172 -78.866);
+
+  --background-color: var(--light-grey);
+  --text-color: var(--black);
+  --link-color: var(--dark-grey);
+
+  --wrapper-width: 1192px;
+
+  &.isAfterDark {
+    --background-color: var(--black);
+    --text-color: var(--off-white);
+    --link-color: var(--grey);
+  }
+}
+
+.newLayout {
+  @mixin body-4;
+  background-color: var(--background-color);
+  color: var(--text-color);
+  width: 100%;
+  padding: 0 20px;
```

Comment:
Because this container gets padding, the footer gets none, which makes the footer very tight on after dark mode: 
<img width="900" alt="image" src="https://user-images.githubusercontent.com/38729/195722383-6311acce-97cb-4bba-90a8-b3b19d422391.png">

Reply:
The issue there is the footer not inheriting the correct colour, will take a look